### PR TITLE
Fix display of uncertain delivery alert on app restart

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-1750

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: Loop/Managers/DeviceDataManager.swift [hunk 1]
```diff
@@ -376,6 +376,14 @@ final class DeviceDataManager {
         return Manager.init(rawState: rawState) as? CGMManagerUI
     }
     
+    func checkDeliveryUncertaintyState() {
+        if let pumpManager = pumpManager, pumpManager.status.deliveryIsUncertain {
+            DispatchQueue.main.async {
+                self.deliveryUncertaintyAlertManager!.showAlert()
```
Comment (rickpasetto) [inline Loop/Managers/DeviceDataManager.swift:382]: [nit] now that this code moved here consider changing

```swift
                self.deliveryUncertaintyAlertManager!.showAlert()
```

to

```swift
                self.deliveryUncertaintyAlertManager?.showAlert()
```

...if only to discourage force-unwraps unless absolutely necessary

**Review by rickpasetto — APPROVED** [review]
one nit, other than that LGTM

File: Loop/Managers/DeviceDataManager.swift (hunk 1 above)
Comment (ps2) [reply Loop/Managers/DeviceDataManager.swift:382]: This one is intentional. `deliveryUncertaintyAlertManager` should only be nil if we do not have a pumpManager.  If we do have a pumpManager, and deliveryStateIsUncertain = true, and we don't have a deliveryUncertaintyAlertManager, that's a failed assertion that should stop the program.

**Pull request merged**